google/mozc
# Add some reading corrections

## Description
A clear and concise description of this pull request.

- 上側: I've misread this word
- 鶴舞公園 & 千種高等学校: famous for misreading names due to JR stations with same kanji notations.

https://www.nhk.or.jp/nagoya/lreport/article/000/31/
https://crd.ndl.go.jp/reference/entry/index.php?id=1000140124&page=ref_view

## Issue IDs
Issue and/or discussion IDs related to this pull request.

## Steps to test new behaviors (if any)
A clear and concise description about how to verify new behaviors (if any).
 - OS: [e.g. Windows 11, macOS 13.1, etc] Any
 - Steps:
   - Suggestion appears when you type "うえがw", "つるまいk", and "ちくさこ" (not yet when "うえが", "つるまい", and "ちくさk")

## Additional context
Add any other context about the pull request here.

![image](https://github.com/user-attachments/assets/48bc712d-3a5e-4a57-890a-0707b5211cf7)

Digital Dai Jisen set the reading of "上側" as "うわ**か**わ" but it hasn't been registered in Mozc's dictionary.
Meikyo Japanese Dictionary and Iwanami Japanese Dictionary adopt "うわがわ".

"千種高校" is registered only in the proprietary Google Japanese Dictionary, so I don't add it.

Disclaimer: I occasionally use Chikusa & Tsurumai stations but am not an OB of Chigusa High School.
